params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
I could, yes, but it would require that everyone using those files follows that convention